json-schema-form.d.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Shorthand to lookup for keys with `x-jsf-*` preffix.
+ */
+export function pickXKey(node: Object, key: 'presentation' | 'errorMessage'): Object | undefined;
+
+type JSConfig = {
```

Comment:
**nit:** Could we rename this type to be something a bit more descriptive? When I first read this, my mind immediately thought "JavaScript" and not "JsonSchema." Maybe `CustomFormConfig` or `CustomJSONConfig`. Thoughts @sandrina-p ?

Reply:
I missed an F there,

In the JSDOC this is called JSFConfig, I don't mind changing the name though I think Config would be enough

Reply:
Yes, let's stick with `JSFConfig`, as it's what we use in multiple files and written docs :)